g11n-ws/modules/md-service-i18n-l2/build.gradle

```diff
@@ -13,7 +13,7 @@ configurations {
 dependencies {
     api project(":vip-common")
     api project(":md-service-i18n-l3")
-    api("com.vmware.singleton:singleton-i18n-patterns-core:+")
+    api("com.vmware.singleton:singleton-i18n-patterns-core:0.5.9")
```

Comment:
This is unnecessary because is't include in #1965 already.

Reply:
Don't worry it, it will auto merge in https://github.com/vmware/singleton/pull/1965